app/controllers/income_details_per_member_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class IncomeDetailsPerMemberController < ManyMemberStepsController
+  private
+
+  def skip?
+    current_snap_application.
+      members.
+      none? { |member| member.employed? || member.self_employed? }
```

Comment:
yeah, we only show questions for employed or self employed people, so this could also be phrased as "if all members are unemployed"